include/libopencm3/stm32/h7/irq.json

```diff
@@ -0,0 +1,155 @@
+{
+    "irqs": [
+        "nvic_wwdg",
+        "pvd",
+        "tamp_stamp",
+        "rtc_wkup",
+        "flash",
+        "rcc",
+        "exti0",
+        "exti1",
+        "exti2",
+        "exti3",
+        "exti4",
+        "dma1_stream0",
+        "dma1_stream1",
+        "dma1_stream2",
+        "dma1_stream3",
+        "dma1_stream4",
+        "dma1_stream5",
+        "dma1_stream6",
```

Comment:
in most other places you seem to be using the acronym column directly, which is good and nice and consistent, why not here? (same for "tamp_stamp")

Reply:
Not sure what happened here. I tried autogenerating parsing the pdf...it obviously didn't go well. I just ran through the requested change. Also did a spot check on the values, especially at the ends of the table and fixed a few other issues.